src/components/FooterBar/FooterBar.tsx

```diff
@@ -84,18 +84,27 @@ const getData = (configFile?: ConfigFile) => {
   return data;
 };
 
+const bottomRender = ({ label, to }: FooterColumnItemProps): React.ReactElement => (
+  <a href={to} className="text-cloud-500 text-sm px-4 border-r">
+    {label}
+  </a>
+);
+
 export const FooterBar: FunctionComponent = () => {
   const { configFile } = usePersistedConfigFile();
   const data = getData(configFile);
+  const networkPath = getNetworkPath(configFile?.network);
+  const legalData = {
+    copyright: "Copyright \u00A9 2021 TradeTrust",
+    items: [
+      { label: "Privacy Policy", to: `${networkPath}/privacy-policy`, render: bottomRender },
+      { label: "Terms of use", to: `${networkPath}/terms-of-use`, render: bottomRender },
+    ],
+  };
```

Comment:
Do you think we could place this piece of data outside of the `FooterBar` component?

Reply:
yup sure